geonode/api/resourcebase_api.py

```diff
@@ -263,6 +267,9 @@ def filter_h_keywords(self, queryset, keywords):
         filtered = queryset.filter(Q(keywords__in=treeqs))
         return filtered
 
+    def x_value_wrapper(self, bbox_x):
```

Comment:
* `bbox_x` isn't a good variable name here (it's a "random" x coordinate, it's irrelevant for the function whether it's from a bbox)

* The function would benefit from a docstring